models/api.models.js

```diff
@@ -0,0 +1,8 @@
+
+const fetchEndPoints = () => {
+    const endPoints = require("../endpoints.json")
+    return endPoints
```

Comment:
I am sure this would work but it does look a little weird. You have gone to the model and required a file inside a function.

Would you need to do this in model at all, Couldn't you just require in the file and use it without going into the model?? Also have a think about the reason you have required a file inside a function, is there a reason you have done it like this as we have never done it like this before?

Reply:
I understand we wouldn't need it in a function as could just export the variable, but as to why its in this file I  wanted to make it consistent with how I my topics endpoint is laid out